mongodb/activity/insertDocument/metadata.go

```diff
@@ -0,0 +1,73 @@
+package insertDocument
+
+import (
+	"github.com/project-flogo/core/data/coerce"
+)
+
+// Settings structure
+type Settings struct {
+	Connection     string `md:"connection,required"`     // The MongoDB connection
+	Operation      string `md:"operation,required"`      // Operation to perform: Insert One Document, Insert Many Documents
```

Comment:
May be add allowed in `md` tag too.

Reply:
Yes, will do

Reply:
Have done this for metadata.go of all activities. Pushed my changes in.